### DISCUSSION TITLE
BCOR auf der Titelseite (Syntaxfehler)

### DISCUSSION THREAD

**Discussion by simsn133** [tudace/tuda_latex_templates#439] [Q&A] [answered]
Hallo an alle,

ich hätte auf der Titelseite gerne eine Bindekorrektur. Eingefügt habe ich sie wie ich es laut Dokumentation verstanden hatte so:

\documentclass[
	...
	BCOR=10mm,%Bindekorrektur, falls notwendig
	BCORtitlepage=true
       ...
          ]{tudapub}

Das führt jedoch bei mir zu einem Error im tudapub.cls, Zeile 196:

! Use of \??? doesn't match its definition.
<argument> \???  
                 ! LaTeX Error: Misplaced '=' in key-value input on line 196
l.196 \Module@Process@SpecialOptions[ptxcd/pub]
                                               
If you say, e.g., `\def\a1{...}', then you must always
put `1' after `\a', since control sequence names are
made up of letters only. The macro here has not been
followed by the required stuff, so I'm ignoring it.

Damit weiß ich leider nichts anzufangen - und die Bindekorrektur erscheint auch nicht wie geplant. Wie benutze ich die Einstellung denn korrekt?

Danke schonmal!

**Comment by TeXhackse** [comment, marked as answer]
Welche version wird verwendet? aktuell sollte das lauten

```
\documentclass[
BCOR=10mm,
BCORtitlepage=true
]{tudapub}

\begin{document}
test
\end{document}
```
tut, was es soll, ggf. fehlt im `…` Bereich ein Komma.

**Reply by simsn133** [reply]
Das Komma war es tatsächlich. Ab und zu ist das Brett vorm Kopf einfach sehr dick. Danke!
